state_of_css.yml

```diff
@@ -537,8 +759,24 @@ translations:
     - key: opinions.sum_up_one_word_css.description
       t: ¿Cómo resumirías tu opinión de CSS en una palabra?
 
+    - key: happiness.pre_post_processors
+      t: ¿Como estás de satisfecho con el estado de los pre/post procesadores?
+    - key: happiness.css_in_js
+      t: ¿Cómo de satisfecho estás con el estado de las librerías de CSS-in-JS?
```

Comment:
Yo pondría: `¿Qué tan satisfecho estás con el estado de las librerías de CSS-in-JS?`

Reply:
El `¿Qué tan satisfecho estás ...?` no es una expresión muy habitual por aquí. Creo que deberíamos mejor mantener `¿Cómo de satisfecho estás con ...?`.